tests/backup/backup_basic_test.go

```diff
@@ -126,6 +123,14 @@ func BackupInitInstance() {
 	kubeconfigList := strings.Split(kubeconfigs, ",")
 	dash.VerifyFatal(len(kubeconfigList), 2, "2 kubeconfigs are required for source and destination cluster")
 	DumpKubeconfigs(kubeconfigList)
+
+	// Switch context to destination cluster to form destination struct containing the required secret/access key, token key, endpoint
+	err = SetDestinationKubeConfig()
+	log.FailOnError(err, "Switching context to destination cluster failed")
```

Comment:
How about we print the rkeMap here  as it might be useful while debugging?

Reply:
done